app/routes/teams.rb

```diff
@@ -156,6 +156,35 @@ class Teams < Core
         end
       end
 
+      get '/teams/:slug/streams/users/:username/tracks/:id' do |slug, username, track_id|
+        please_login
+        only_with_existing_team(slug) do |team|
+          unless team.includes?(current_user)
+            flash[:error] = "You may only view team pages for teams that you are a member of, or that you manage."
+            redirect '/'
+          end
+
+          user = ::User.find_by_username(username)
+          stream = TeamStream.new(team, current_user.id, params['page'] || 1)
+          stream.track_id = track_id.downcase
```

Comment:
No need to `downcase` here, the `writer` already [takes care of it](https://github.com/exercism/exercism.io/blob/master/lib/exercism/team_stream.rb#L19).